app/templates/components/ui-table/cell/cell-sessions-dashboard.hbs

```diff
@@ -1,8 +1,8 @@
 <div class="ui list">
-  <div class="item">{{t 'Draft'}}: {{record.eventStatisticsGeneral.sessionsDraft}}</div>
-  <div class="item">{{t 'Submitted'}}: {{record.eventStatisticsGeneral.sessionsSubmitted}}</div>
-  <div class="item">{{t 'Accepted'}}: {{record.eventStatisticsGeneral.sessionsAccepted}}</div>
-  <div class="item">{{t 'Confirmed'}}: {{record.eventStatisticsGeneral.sessionsConfirmed}}</div>
-  <div class="item">{{t 'Pending'}}: {{record.eventStatisticsGeneral.sessionsPending}}</div>
-  <div class="item">{{t 'Rejected'}}: {{record.eventStatisticsGeneral.sessionsRejected}}</div>
+  <div class="item">1. {{t 'Draft'}}: {{record.eventStatisticsGeneral.sessionsDraft}}</div>
+  <div class="item">2. {{t 'Submitted'}}: {{record.eventStatisticsGeneral.sessionsSubmitted}}</div>
+  <div class="item">3. {{t 'Accepted'}}: {{record.eventStatisticsGeneral.sessionsAccepted}}</div>
+  <div class="item">4. {{t 'Confirmed'}}: {{record.eventStatisticsGeneral.sessionsConfirmed}}</div>
+  <div class="item">5. {{t 'Pending'}}: {{record.eventStatisticsGeneral.sessionsPending}}</div>
+  <div class="item">6. {{t 'Rejected'}}: {{record.eventStatisticsGeneral.sessionsRejected}}</div>
```

Comment:
@harsh-jindal Kindly refer to this issue #1809 . This particular file change is simply reintroducing the potential space wastage and dis-alignment. So i'd suggest that you remove the numbering

Reply:
@uds5501 I'll make the changes.